docs/guides/recipe.rst

```diff
@@ -69,21 +74,52 @@ non-executable fields of ``package.json`` such as ``name``, ``version``,
 * **keywords**: A list of strings that are related to the package.
 * **maintainers**: A list of ``Maintainer`` objects that describe the contact
```

Comment:
We might want to link the objects like Maintainer to its schema so one can see how to create one: https://pollination.github.io/queenbee/_static/redoc-recipe.html#tag/maintainer_model